NodeJS/__integration__/RestClientAgent.it.ts

```diff
@@ -94,4 +101,36 @@ describe("RestClient", () => {
         let submitted = await restClient.requestAgentDeletion(username, "7300000100")
         expect(submitted).toBeTruthy()
      })
+
+     test("createSkillJob with unavailable extension number should fail", async () => {
+         let jobRequest:SkillCreateRequest = {
+             name: 'survey',
+             number: 1030,
+             clientId: 1,
+             skillType: 'AGENT',
+             acwInterval: null,
+             slaInSeconds: null,
+             slaPercentage: null,
+             announcementExtension: null
+         }
+         let submitted = await restClient.createSkillJob(jobRequest)
+         expect(submitted).toEqual(-500)
+     })
+     /*
+     number needs to be in reserved state for this to work.
+     */
+     test("createSkillJob with reserved extension should work.", async () => {
+        let jobRequest:SkillCreateRequest = {
+            name: 'survey',
+            number: 103,
```

Comment:
Instead of having a comment, I'd use the code to tell that story. eg:

```
const reservedNumber: number = 103
const jobRequest:SkillCreateRequest = {
            /* ... */
            number: reservedNumber,
            /* ... */
}
```

Reply:
good idea.